sdk/formrecognizer/ai-form-recognizer/test/public/node/formrecognizerclient.spec.ts

```diff
@@ -152,12 +160,11 @@ matrix([[true, false]] as const, async (useAad) => {
 
         assert.ok(page);
 
-        /* TODO: layout regression
-          assert.isNotEmpty(page.tables);
-          const [table] = page.tables!;
-          assert.ok(table.boundingBox);
-          assert.equal(table.pageNumber, 1);
-          */
+        // TODO: layout regression
```

Comment:
is this still a regression? I don't think I see this in .NET

Reply:
These still throw when I uncomment them. There are several other things that i reenabled, but I still don't get a table on the first page of this document.